### DISCUSSION TITLE
Some ideas for this spectacular theme

### DISCUSSION THREAD

**Discussion by gabrielgbs97** [nunocoracao/blowfish#98] [Ideas]
First of all, I would like to thank the main developer @nunocoracao of this theme and upstream theme [Congo](https://github.com/jpanther/Congo). Here are some ideas.
1.- Include Ko-Fi, buymeacoffe, and other donation like sites as social media links.
2.- Commends section may be appreciated for blog posts. This has to be handled by a 3rd party service. Developing a tailored solution (i.e. Firebase) would be a bit over-kill. A good option would be Disqus. https://gohugo.io/content-management/comments/

**Comment by nunocoracao** [comment]
Thanks @gabrielgbs97, for the kudos and for the ideas. A couple of comments/questions so that we can move forward. 

1- those sound like an interesting addition. It touches a little bit with another thread I opened regarding integrations. I feel like that could provide value to the theme users. In order to move forward, do you want to open the issues for each integration or would you rather me do it? 

2- I thought about making something similar to views and likes (which already integrate into firebase) so it would be easy to have a comment system on top of a firebase section. Having said this, it would be a massive effort to make sure each site owner can control safety measures, block words, etc. Integrations with Disqus are already pretty well described, do you think streamlining those would provide value?

**Comment by gabrielgbs97** [comment]
1.- I think I can integrate it when I have some spare time as a social media icon. If a more independent feature is needed would be more complicated for me.
3.- I think it would be simpler for beginners to wrap Disqus functionality like `enable_disqus_on=["single"]` . The default `disqusShortname` is still needed. Or even simpler, when disqusShortname is detected, put internal template automatically on posts pages.

For now, user would have to maintain a separate copy in `/layouts/whatever.html` to override a type of page. Eventually breaking something or loosing functionality when a theme update happens. I am a Hugo newbie, may be this has an easier path?

**Reply by gabrielgbs97** [reply]
ko-fi icon: #100 Added and waiting for approval

**Reply by nunocoracao** [reply]
@gabrielgbs97 approved
